java/java-tests/testSrc/com/intellij/java/refactoring/AnonymousToInnerTest.java

```diff
@@ -133,7 +133,18 @@ protected boolean showRefactoringDialog() {
   public void testTypeParameterNotMentioned() {
     doTest("MyClass", true);
   }
-  
+
+  public void testAddThisQualifier() {
+    doTest("MyClass", true);
+  }
+
+  public void testAddThisQualifierWithImplicitThis() {
+    doTest("MyClass", true);
+  }
+
+  public void testOuterThisQualifier() {
+    doTest("MyClass", true);
+  }
```

Comment:
The first two tests are already passing on latest IJ 2025.1. But the last one (`outerThisQualifier`) fails.

(just a note, no action required here)

Reply:
Yep, I was just adding more test coverage to make sure I didn't break anything